server/models/Profile.js

```diff
@@ -0,0 +1,51 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema({
+  email: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+  },
+  city: {
+    type: String,
+    default: "",
+  },
+  phone: {
+    type: Number,
+  },
+  address: {
+    type: String,
+  },
+  photo: {
+    type: String,
+  },
+  availability: {
```

Comment:
This design for availability won't fit the use cases of our project but let's focus on it at a later date